packages/latex/src/main.rs

```diff
@@ -156,6 +156,7 @@ fn escape_text(module: Value) -> String {
         let s = s.split('\\').map(|t| t.replace('{', r"\\{").replace('}', r"\\}")).collect::<Vec<String>>().join(r"\\textbacklash{}")
             .replace('#', r"\\#")
             .replace('$', r"\\$")
+            .replace("\"","\\\"")
```

Comment:
This makes it seem like you are escaping the LaTeX character, which isn't what you actually are doing. You are adding a backslash behind the quote to escape the JSON formatting, which doesn't have anything to do with escaping " in LaTeX. See line 168, that's where the bug originates from. You are just inserting the string `s` into the JSON, which makes you need to escape all backslashes in this and all other replacements. I think this is bad and that it would be much better to use serde_json to serialize the JSON, so it is clear what you are doing so others understand easier and don't make the same mistake. You can, for example, use the json! macro to generate the json for you. Also, you having to replace each newline with a space is also due to the json not being serialized properly, and fixing that would make the output more alike the input, and look better.